frontend/viewer/src/project/browse/SearchFilter.svelte

```diff
@@ -78,6 +125,30 @@
     </ComposableInput>
   </div>
   <Collapsible.Content class="p-2 mb-2 space-y-2">
+    <div class="flex flex-col @md/list:flex-row gap-2 items-stretch">
+      <div class="flex flex-row gap-2 flex-1">
+        <!-- Field Picker -->
+        <Select
+          bind:value={selectedField}
+          options={fields}
+          idSelector="id"
+          labelSelector="label"
+          placeholder={$t`Field`}
+          class="flex-1"
+        />
+        <!-- Writing System Picker -->
+        <WsSelect bind:value={selectedWs} wsType={selectedField.ws} />
+      </div>
+      <!-- Text Box: on mobile, wraps to new line -->
+      <div class="flex flex-row gap-2 flex-1">
+        <OpFilter bind:value={filterOp}/>
+        <Input
+          bind:value={fieldFilterValue}
+          placeholder={$t`Filter for`}
+          class="flex-1"
+        />
+      </div>
+    </div>
     <Switch bind:checked={missingExamples} label={$t`Missing Examples`} />
     <Switch bind:checked={missingSenses} label={$t`Missing Senses`} />
     <Switch bind:checked={missingPartOfSpeech} label={$t`Missing Part of Speech`} />
```

Comment:
Minor UX issue: I opened the writing systems Select and selected some wses, then clicked outside the dropdown to close it. I chose an area of the dialog that was blank, but in fact my click landed in the `<div>` of one of these Switch components and it was turned on. Not a big deal, I could turn it off, but it might be nice if the clickable area of the Switch components was not their entire div (which extends all the way to the edge of the dialog) but only the portions that contain the switch icon and text.

Reply:
Yeah that sounds like something to fix later, I don't really like those filters anyway so they'll probably go away at some point, or maybe turn into a dropdown as turning them all on doesn't make much sense